src/core/database/database.ts

```diff
@@ -372,6 +433,14 @@ export class DatabaseManager {
         CREATE UNIQUE INDEX IF NOT EXISTS idx_identify_provider_id ON _identifies(provider, provider_id);
         CREATE INDEX IF NOT EXISTS idx_edge_functions_slug ON _edge_functions(slug);
         CREATE INDEX IF NOT EXISTS idx_edge_functions_status ON _edge_functions(status);
+        
+        -- Better Auth indexes
+        CREATE INDEX IF NOT EXISTS idx_user_email ON "user"("email");
+        CREATE INDEX IF NOT EXISTS idx_user_role ON "user"("role");
```

Comment:
Since almost all user has the role of authenticated, idx_user_role doesn't work well...